L1Trigger/L1TMuonEndCap/src/bdt/Forest.cc

```diff
@@ -8,7 +8,7 @@
 //     Tibshirani, and Friedman.                                        //
 //    *Greedy Function Approximation: A Gradient Boosting Machine.      //
 //     Friedman. The Annals of Statistics, Vol. 29, No. 5. Oct 2001.    //
```

Comment:
Leave file as is.

Reply:
Since there's no changes except spaces, it should be trivial to review. I'd really prefer to have no trailing spaces and tabs.